src/mailingui/components/buttons/Buttons.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC, ReactNode, CSSProperties } from "react";
+import { Button as ReactEmailButton } from "@react-email/components";
+
+interface ButtonProps {
+  paddingX?: number;
+  paddingY?: number;
+  href: string;
+  children: ReactNode;
+  backgroundColor?: CSSProperties["color"];
+  fontSize?: CSSProperties["fontSize"];
+  color?: CSSProperties["color"];
+  borderColor?: CSSProperties["color"];
+  pill?: boolean;
```

Comment:
I'm not quite sure whether I like this way of having `pill` as a property. 

What do you guys think if we instead provide some enum for `rounded` property?

1. I had to google what do we mean by `pill`, it was not very obvious
2. If we want to have let's say 3 versions of how rounded borders we will have, it won't be possible.

WDYT @iamhectorsosa @ericvalcik ?

Reply:
I think both can work. I have no problem with changing it to `rounded` with an enum with values like `none`, `sm`, `md`, `lg` `full` for example.

I agree with the proposal

Reply:
I like the change @ericvalcik did